flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
what't the rationale for excluding aarch64?

Reply:
Airshipper downloads binaries of Voxygen, the client to play Veloren. At the moment, those builds are only available for x86_64.